pkg/info/info_test.go

```diff
@@ -117,6 +92,7 @@ func TestLoadManifestFromReceiptOrIndex(t *testing.T) {
 			} else {
 				if err != nil {
 					t.Error(err)
+					return
```

Comment:
maybe use `t.Fatal(err)` instead?

Reply:
since this is a for-loop with test cases, it's not a great idea. if we Fatal, other cases won't run.